pytensor/xtensor/type.py

```diff
@@ -516,6 +578,48 @@ def set(self, value):
         return px.indexing.index_assignment(x, value, *idxs)
 
     def inc(self, value):
+        """Return a copy of the variable indexed by self with the indexed values incremented by value.
+
+                The original variable is not modified.
+
+        Raises
+        ------
+        Value:
+            If self is not the result of an index operation
+
+        Examples
+        --------
+
+        .. test-code::
```

Comment:
```suggestion
        .. testcode::
```

None of these blocks are being shown in the rendered version: https://pytensor--1533.org.readthedocs.build/en/1533/library/xtensor/type.html#pytensor.xtensor.type.XTensorVariable.inc

I think it is because the directive name is without hyphen: https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testcode

Reply:
fixed